From 814be24663d089d8f9db655ae7e488c4a6de0851 Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Fri, 7 Aug 2015 12:39:09 +0000
Subject: [PATCH] fix-fortify-system-wrappers.patch

The wrapper features.h gets pulled in by system headers causing things to
break. We work around it by simply not wrap features.h
---
 config/system-headers.mozbuild | 1 -
 1 file changed, 1 deletion(-)

diff --git a/config/system-headers.mozbuild b/config/system-headers.mozbuild
index b8c14159a7c50..1421bf12bde1d 100644
--- a/config/system-headers.mozbuild
+++ b/config/system-headers.mozbuild
@@ -222,7 +222,6 @@ system_headers = [
     "exe386.h",
     "extras.h",
     "fcntl.h",
-    "features.h",
     "fenv.h",
     "ffi.h",
     "fibdef.h",
-- 
2.56.0

